### PR TITLE
News about OSS contributions

### PR DESCRIPTION
**Pull request by mccalluc**
Are the descriptions of your work accurate? Is my example too trivial? Anyone else in the lab we should nudge to add a line?

### CODE REVIEW

File: _news/2020-11-01-os-contributions.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Our work in the Gehlenborg lab depends on open source software, but we also give back to the community,
+and are contributors to several projects.
+- Ilan Gold [extended Deck.gl's tiling](https://github.com/visgl/deck.gl/pulls?q=is%3Apr+author%3Ailan-gold)
+  to work for non-geographic tile sets,
+  and Viv is now featured in the [Deck.gl showcase](https://deck.gl/showcase)
+- Trevor Manz [presented](https://forum.image.sc/t/upcoming-calls-on-next-gen-bioimaging-data-tools-starting-oct-29/43489/34)
+  work on Viv recently at an OME meeting on next-generation file formats (NGFF),
+  and has made a [lot of PRs to zarr.js](https://github.com/gzuidhof/zarr.js/pulls?q=is%3Apr+author%3Amanzt).
+- Open source isn't just about code. Chuck can be nit-picky about grammar, and at one point fixed
+  [dozens](https://github.com/cypress-io/cypress-documentation/pull/195/files) of it's/its mistakes in the cypress.io docs.
```
Comment (ilan-gold) [inline _news/2020-11-01-os-contributions.md:20]: Not too trivial at all :)

Comment (ilan-gold) [reply _news/2020-11-01-os-contributions.md:20]: Also deck.gl! https://github.com/visgl/deck.gl/pulls?q=is%3Apr+author%3Amccalluc

**Review by ilan-gold — APPROVED** [review]

**Review by manzt — APPROVED** [review]
Great, thanks for writing this up @mccalluc!

**Comment by mccalluc** [comment]
@ngehlenborg : Tone ok? It felt like there was something worth advertising here that cut across projects. Do you know if there's anyone else in the lab we should nudge?

File: _news/2020-11-01-os-contributions.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+---
+Our work in the Gehlenborg lab depends on open source software, but we also give back to the community,
+and are contributors to several projects.
+- Ilan Gold [extended Deck.gl's tiling](https://github.com/visgl/deck.gl/pulls?q=is%3Apr+author%3Ailan-gold)
+  to work for non-geographic tile sets,
+  and Viv is now featured in the [Deck.gl showcase](https://deck.gl/showcase).
+- Trevor Manz [presented](https://forum.image.sc/t/upcoming-calls-on-next-gen-bioimaging-data-tools-starting-oct-29/43489/34)
+  work on Viv recently at an OME meeting on next-generation file formats (NGFF),
+  and has made a [lot of PRs to zarr.js](https://github.com/gzuidhof/zarr.js/pulls?q=is%3Apr+author%3Amanzt).
+- Open source isn't just about code. Chuck can be nit-picky about grammar,
```
Comment (ngehlenborg) [inline _news/2020-11-01-os-contributions.md:19]: Can we change that to "Open source isn't just about code. Chuck McCallum has contributed to the documentation of [Cypress.io](https://github.com/cypress-io/cypress-documentation/pull/195/files) and [Deck.gl](https://github.com/visgl/deck.gl/pulls?q=is%3Apr+author%3Amccalluc)".

**Review by ngehlenborg — CHANGES_REQUESTED** [review]

File: _news/2020-11-01-os-contributions.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  and has fixed [it's/its mistakes](https://github.com/cypress-io/cypress-documentation/pull/195/files)
+  in the cypress.io docs and 
+  [typos](https://github.com/visgl/deck.gl/pulls?q=is%3Apr+author%3Amccalluc) in the deck.gl docs.
```
Comment (mccalluc) [inline _news/2020-11-01-os-contributions.md:22]:
```suggestion
- Open source isn't just about code. Chuck McCallum has contributed to the documentation of
  [Cypress.io](https://github.com/cypress-io/cypress-documentation/pull/195/files)
  and [Deck.gl](https://github.com/visgl/deck.gl/pulls?q=is%3Apr+author%3Amccalluc).
```

**Pull request merged**